README.md

```diff
@@ -1,2 +1,18 @@
-# training-operator
-Kubeflow Training Operator
+# Training Operator
+
+## Overview
+
+This repository hosts the Kubernetes Training Operator for Kubeflow training jobs.
+
+## Description
+
+The [Kubeflow Training Operator][1] provides Kubernetes custom resources to run distributed or non-distributed training through so called training jobs, such as TFJobs and PytorchJobs. The Training Operator in this repository is a Python script that wraps the latest released Kubeflow Training Operator [manifests][2], providing lifecycle management and handling events (install, upgrade, integrate, remove). It is part of the [Charmed Kubeflow][3] operators.
+
+## Usage
+
+While it is possible to deploy the Training Operator as a standalone operator, it works best when deployed alongside other components included in the Kubeflow bundle. For installation steps, please refer to the [installation][4] guide.
+
+[1]: https://www.kubeflow.org/docs/components/training/
```

Comment:
These are fine. Usually in docs we use the implicit reference.  so we would put `[docs][]` in the text and then the reference is just `[docs]: http://whatever`. It hardly matters here, but in larger docs it makes it much easier to correlate the links when/if they need updating

Reply:
Noted